shepherd_bms/src/stateMachine.cpp

```diff
@@ -31,7 +41,10 @@ void StateMachine::handleReady(AccumulatorData_t *bmsdata)
 
 void StateMachine::initCharging()
 {
-    return;
+    compute.enableCharging(true);
+    segment.enableBalancing(false);
+    static Timer chargeMessageTimer;
+	static const uint16_t CHARGE_MESSAGE_WAIT = 250; //ms
```

Comment:
same as above, I dont think we want to be declaring anything in these functions. Maybe define them outside and just call/use them as needed here